catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/impl/CachedResourceMetacardComparator.java

```diff
@@ -50,28 +46,7 @@
       LoggerFactory.getLogger(CachedResourceMetacardComparator.class);
 
   private static final List<Function<Metacard, ?>> METACARD_METHODS =
-      ImmutableList.of(
-          Metacard::getModifiedDate,
-          Metacard::getThumbnail,
-          Metacard::getResourceURI,
-          Metacard::getLocation,
-          Metacard::getEffectiveDate,
-          Metacard::getCreatedDate,
-          Metacard::getExpirationDate,
-          Metacard::getContentTypeName);
-
-  private static final Set<String> ATTRIBUTES_TO_IGNORE =
-      ImmutableSet.of(
-          Metacard.CHECKSUM,
-          Metacard.MODIFIED,
-          Metacard.RESOURCE_SIZE,
-          Metacard.THUMBNAIL,
-          Metacard.RESOURCE_URI,
-          Metacard.GEOGRAPHY,
-          Metacard.EFFECTIVE,
-          Metacard.CREATED,
-          Metacard.EXPIRATION,
-          Metacard.CONTENT_TYPE);
+      ImmutableList.of(Metacard::getModifiedDate, Metacard::getCreatedDate);
```

Comment:
❗️ Since we've changed how the comparison is done, we should update the JavaDoc on the `isSame()` method.

Reply:
fixed